vaadin-testbench-core-junit5/src/test/java/com/vaadin/testbench/browser/BrowserHubTest.java

```diff
@@ -43,4 +45,37 @@ public void hubFromAnnotationOrSystemProperty() {
         }
     }
 
+    @Test
+    public void hubPortFromDefaultValueOrParametersSetter() {
+        String oldPortProperty = System.getProperty(HUB_PORT_PROPERTY);
+        String oldHostnameProperty = System.getProperty(HUB_HOSTNAME_PROPERTY);
+
+        try {
+            // Reset the hub hostname property (removes the saucelab url in CI)
+            System.clearProperty(HUB_HOSTNAME_PROPERTY);
+
+            // Default must be the "official" 4444 port for backwards
+            // compatibility
+            Assertions.assertEquals(getExpectedHubUrl(4444),
```

Comment:
Something is off with the test on CI server as it gets the url wrong ` <http://hub-in-annotation:4444/wd/hub> but was: <https://ondemand.us-west-1.saucelabs.com/wd/hub>` and it's on the default value.

Reply:
Yep, this is what Marco told about https://github.com/vaadin/testbench/pull/1658#discussion_r1286819490 . This was overlapped with other errors. Will see what I can do with this.